test/e2e/run_test.go

```diff
@@ -1376,7 +1376,27 @@ USER mail`
 	})
 
 	It("podman run --tz", func() {
-		session := podmanTest.Podman([]string{"run", "--tz", "foo", "--rm", ALPINE, "date"})
+		testDir := filepath.Join(podmanTest.RunRoot, "tz-test")
+		err := os.MkdirAll(testDir, 0755)
+		Expect(err).To(BeNil())
+
+		tzFile := filepath.Join(testdir, "tzfile.txt")
+		file, err := os.Create(tzFile)
+		Expect(err).To(BeNil())
+		defer file.Close()
+
+		_, err = file.WriteString("Hello")
+		Expect(err).To(BeNil())
+
+		session := podmanTest.Podman([]string{"run", "--tz", ("../../../" + tzFile), "--rm", ALPINE, "date"})
```

Comment:
Doesn't Go have some sort of path joiner? Would it make sense to use that instead of string concatenation?

Reply:
7 actually, this is the cheap and cheezy one.  I'll work up another approach